### PR TITLE
Adds prisoner job files

### PR DESCRIPTION
**Pull request by vltevev**
Adds the files for a new position on the station, the prisoner. The prisoner spawns in permanent confinement and primary duty is to keep the warden occupied. They serve no real purpose outside of arrpee.

This PR only includes the job files, seeing as some large map changes are coming soon and I don't want to delay any merges. 

[gameplay]
:cl:
 * rscadd: Adds files for a new prisoner job. (not currently playable.)

### CODE REVIEW

File: code/game/jobs/job/whitelisted.dm [hunk 1]
```diff
@@ -89,3 +89,55 @@
...
+		if(4)
+			H.equip_or_collect(new/obj/item/weapon/storage/backpack/satchel(H), slot_back)
+		if(5)
+			H.equip_or_collect(new /obj/item/weapon/storage/backpack/messenger(H), slot_back)
+
+	return 1
+
+/datum/job/prisoner/introduce(mob/living/carbon/human/M, job_title)
+	if(!job_title)
+		job_title = src.title
```
Comment (ghost) [inline code/game/jobs/job/whitelisted.dm]: I know this is just copypasted but the src is unnecessary.

**Comment by DrSnips** [comment]
My only concern is a nagging feeling that this is just going to be a pool of grug tier assholes just looking to prison break asap.

File: code/game/jobs/job/whitelisted.dm (hunk 1 above)
Comment (vltevev) [reply code/game/jobs/job/whitelisted.dm]: Should I remove those three lines then?

**Comment by yackemflam** [comment]
damn it, I was working on that

File: code/game/jobs/job/whitelisted.dm (hunk 1 above)
Comment (ghost) [reply code/game/jobs/job/whitelisted.dm]: Just the `src.` bit, it's redundant since `title` is already var of `/datum/job`

**Comment by TGFH** [comment]
I don't see the benefit in adding a role like this since it will probably attract people looking for a license to tide. 
Is the prisoner subject to the same rules and potential punishments that a regular inmate would be (Ie: Getting turned horizontal for a breakout)? How exactly are they going to "Keep the warden occupied" when the warden doesn't really need to interact with a perma inmate at all? Not to mention that the warden has an actual role to fulfill so if they are busy doing their real job then the inmate will quickly find themselves becoming lonely and bored/
I get that the idea is to create an RP focused role, but for this role to work it would require both the inmate and the warden to play ball. If the warden isn't interested (or not present) then the inmate's only viable options boil down to going braindead, breaking out (and hoping someone actually cares), or simply asking to be released and getting the HoP to give him a new job.

**Comment by cloutcode** [comment]
i like the idea of a prisoner and a warden having a bond you don't get with normal crewmembers that either 1. ghost 2. afk 3. break out for their sentences

**Comment by FudgePucker** [comment]
Would need a pop up or giant red text in the chat box that tells 'THIS IS A TURBO RP ROLE, NOT A LICENSE TO BE A SHITTER.' So that people understand that taking this role just to have the AI break them into the armory is a bannable offence. Also require a warden or three+ security members so that you arent just stuck in an unpowered prison cell alone. Do this and it's a :+1: from me

**Comment by Goldsteel** [comment]
Make prisoner transfer a random event.

**Comment by yackemflam** [comment]
Why not both?

File: code/game/jobs/job/whitelisted.dm (hunk 1 above)
Comment (vltevev) [reply code/game/jobs/job/whitelisted.dm]: Done and done.

**Comment by PrimeDSS13** [comment]
ehhhh don't really see the point

**Comment by gbasood** [comment]
> Adds files for a new prisoner job. (not currently playable.)

As anything but a round-start role, which this currently is not, this is probably fine. I'm not sure if you'd see it show up in a round because of that, though.

**Comment by vltevev** [comment]
I'll close this and work on a more interesting implementation that doesn't just boil down to 'keep the warden from going braindead'.

**Pull request closed without merging**